views/partials/nav.ejs

```diff
@@ -54,11 +54,6 @@
 
   <!-- <label>Misc.</label> -->
   <ul class="nav nav-sidebar">
-    <li>
```

Comment:
Is this just not wired up?

Reply:
@Avuxo moved this to the test-bench-utils and forget to remove the link in the nav for the one off route